salt/utils/thin.py

```diff
@@ -381,7 +389,7 @@ def _get_supported_py_config(tops, extended_cfg):
     for the supported Python interpreter versions. This is then written into the thin.tgz
     archive and then verified by salt.client.ssh.ssh_py_shim.get_executable()
 
-    Note: Minimum default of 2.x versions is 2.7 and 3.x is 3.0, unless specified in namespaces.
+    Note: Minimum default of 3.x is 3.0, unless specified in namespaces.
```

Comment:
Are we sure on 3.0, salt is only meant to work with 3.5, right?
Or for this piece of code it doesn't really matter?

Reply:
I will re-word this. You can even actually use 2.7 if using an older version of salt.